modules/tinymce/src/core/main/ts/annotate/Identification.ts

```diff
@@ -23,24 +23,17 @@ const identify = (editor: Editor, annotationName: Optional<string>): Optional<{
   const newStart = Traverse.child(start, rng.startOffset).getOr(start);
   const closest = SelectorFind.closest(newStart, selector, isRoot(root));
 
-  const getAttr = (c, property: string): Optional<any> => {
-    if (Attribute.has(c, property)) {
-      return Optional.some(Attribute.get(c, property));
-    } else {
-      return Optional.none();
-    }
-  };
-
-  return closest.bind((c) => getAttr(c, `${Markings.dataAnnotationId()}`).bind((uid) =>
-    getAttr(c, `${Markings.dataAnnotation()}`).map((name) => {
-      const elements = findMarkers(editor, uid);
-      return {
-        uid,
-        name,
-        elements
-      };
-    })
-  ));
+  return closest.bind((c) =>
+    Attribute.getOpt(c, `${Markings.dataAnnotationId()}`).bind((uid) =>
+      Attribute.getOpt(c, `${Markings.dataAnnotation()}`).map((name) => {
+        const elements = findMarkers(editor, uid);
+        return {
+          uid,
+          name,
+          elements
+        };
+      })
+    ));
```

Comment:
This is just a simplified version to avoid a nullable return value from `Attribute.get`. It should be equivalent though, as doing `has` and `get` should be the same as using an `Optional`.